nvtabular/workflow.py

```diff
@@ -65,17 +65,7 @@ class BaseWorkflow:
         Names of the continous columns.
     label_name : list of str
         Names of the label column.
-    feat_ops : list of feature engineering operator objects
-        Feature engineering ops objects GroupBy.
-        New feature engineering operators can be added
-        later using the other functions
-    stat_ops : list of stats operator objects
-        Feature engineering ops objects such as
-        Categorify and Normalize. New feature engineering
-        operators can be added later using the other functions.
-    df_ops : list of data frame operator objects
-    to_cpu : bool, default True
-    config : bool
+    config : object
```

Comment:
Maybe we can explain the `config` argument a bit more here?

Reply:
I think we should actually make config a class in its own right (like Alec suggested here https://github.com/NVIDIA/NVTabular/issues/15).